app.py

```diff
@@ -21,41 +22,53 @@
     authority=app.config.get("AUTHORITY"),
     client_id=app.config["CLIENT_ID"],
     client_credential=app.config["CLIENT_SECRET"],
-    )
+)
+
 
 @app.route("/login")
 def login():
-    return render_template("login.html", version=identity.__version__, **auth.log_in(
-        scopes=app_config.SCOPE,  # Have user consent scopes during log-in
-        redirect_uri=url_for("auth_response", _external=True),  # Optional. If present, this absolute URL must match your app's redirect_uri registered in Azure Portal
-        ))
+    result = auth.log_in(
+        # Have user consent scopes during log-in
+        scopes=app_config.SCOPE,
+        # Optional. If present, this absolute URL must match your app's redirect_uri registered in Azure Portal
+        redirect_uri=url_for("auth_response", _external=True),
+    )
```

Comment:
Some of these lines were quite long (longer even than 127, which is my generous line-length cutoff), so I reorganized the code to put comments on their own lines.

Reply:
FWIW, I do make an effort to keep line length reasonably close to PEP8's 79 recommendation, typically maxing out at around 89-ish. It is just that I happen to also prefer inline comment style (demonstrated in [this answer](https://stackoverflow.com/a/13761377)) so I have to come up with a convention that "inline comments do not count into line length". :-)

You can keep your changes here.